zenodo/config.py

```diff
@@ -1265,11 +1265,6 @@ def safelist_sort(_):
         'schema': 'http://schema.datacite.org/meta/kernel-4.1/metadata.xsd',
         'serializer': 'zenodo.modules.records.serializers.oaipmh_datacite_v41',
     },
-    'datacite3': {
```

Comment:
I missed on below `oai_datacite3`, and the `oai_datacite should have the same config as `oai_datacite4`